server/index.js

```diff
@@ -0,0 +1,231 @@
+const express = require("express");
+const cors = require("cors");
+const mysql = require("mysql2");
+const joi = require("joi");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const jwt_decode = require("jwt-decode");
+const { authenticate } = require("./middleware");
+
+require("dotenv").config();
+
+const server = express();
+server.use(express.json());
+server.use(cors());
+
+const mysqlConfig = {
+  host: "localhost",
+  user: "root",
+  password: process.env.DB_PASS,
+  database: "bill_app",
+};
+
+const userSchema = joi.object({
+  full_name: joi.string().trim().min(2).required(),
+  email: joi.string().email().trim().lowercase().required(),
+  password: joi.string().required(),
+});
+const dbPool = mysql.createPool(mysqlConfig).promise();
+
+server.get("/groups", authenticate, async (req, res) => {
+  console.log(req.user);
+
+  try {
+    const [data] = await dbPool.execute(
+      `
+      SELECT * FROM  \`groups\`
+    `,
+    );
+    return res.status(201).send({ data });
```

Comment:
201 status code for when we create entities.
Here you are getting entities so the correct status code would be 200